src/translations/fr-FR.json

```diff
@@ -166,6 +167,6 @@
     "mvf.operator.notBetween": "non compris entre",
     "mvf.operator.between.tooltip.bubble": "Les valeurs de début et de fin sont incluses.",
     "mvf.operator.notBetween.tooltip.bubble": "Les valeurs de début et de fin ne sont pas incluses.",
-    "mvf.treatNullValuesAsZeroLabel": "Traiter les valeurs vides en tant que 0",
+    "mvf.treatNullValuesAsZeroLabel": "Traiter les valeurs vides comme 0",
```

Comment:
Bugfix related to https://jira.intgdc.com/browse/BB-2235

Reply:
I tested the new string and it fixes the mentioned bug.